tests/test_block_creation_mvp.py

```diff
@@ -225,3 +225,27 @@ def test_transaction_wrong_payload(self, transaction_template, data_set):
             )
         except TypeError as e:
             assert str(e) == data_set["error"]
+
+    @pytest.mark.parametrize("transaction_template",
+                             [TransactionTemplates.legacy_transaction])
+    def test_replace_transaction(self, transaction_template):
+
+        transaction_template_low_gas_price = transaction_template(self.web3,
+                                                                  self.receiver_address,
+                                                                  self.sender_private_key)
+
+        transaction_template_low_gas_price.update(
+            {"gasPrice": int(self.web3.eth.gas_price / 1000)}
+        )
+        low_gas_price_txn_hash = self.send_transaction(transaction_template_low_gas_price)
+        low_gas_price_txn_nonce = self.web3.eth.get_transaction(low_gas_price_txn_hash)["nonce"]
+
+        assert low_gas_price_txn_hash in str(self.web3.geth.txpool.content())
+
+        transaction_template_low_gas_price.update(
+            {"gasPrice": int(self.web3.eth.gas_price * 3),
+             "nonce": low_gas_price_txn_nonce})
+
+        replacing_txn_hash = self.send_transaction(transaction_template_low_gas_price)
+
+        assert replacing_txn_hash in str(self.web3.geth.txpool.content())
```

Comment:
We need to check that first transaction (low_gas_price_txn_hash) is not in the pool now (replaced).
And, replacing_txn_hash - should be mined and included into block. So let's check atthe end that replacing_txn_hash is mined and no ((1) low_gas_price_txn_hash AND (2) replacing_txn_hash) in the tx_pool